Docs/Book/GettingStartedInPython.rst

```diff
@@ -3259,6 +3259,176 @@ out the code will automatically assign labels:
+and will most likely be between 2 and 4.  There is not a maximum on the number of bonds used in the cutting, because
+2 non-ring bonds will give rise to 3 fragments, but 4 ring bonds may also do so, 2 splitting one ring and the other 2
+a different ring in the query.
+This produces a large number of fragment sets, where each set is produced by cutting one particular combination
+of bonds.  Each fragment in the set is compared with synthons in one column of the synthons for each reaction
+thus identifying combinations of synthons that might match the query.  These combinations are enumerated and
+a final comparison with the query performed to select the hit set.  The algorithm is largely similar to that
+described by `Liphardt and Sander <https://doi.org/10.1021/acs.jcim.3c00290>`_.
+
+Database Preparation
+====================
+
+The databases are normally supplied in text form, and can be used as such:
+
+.. doctest::
+
+  >>> from rdkit import Chem
+  >>> from rdkit.Chem import rdSynthonSpaceSearch, rdFingerprintGenerator
+  >>> import os
+  >>> RDBASE = os.environ["RDBASE"]
+  >>> synthonspace = rdSynthonSpaceSearch.SynthonSpace()
+  >>> textFile = f"{RDBASE}/Code/GraphMol/SynthonSpaceSearch/data/amide_space.txt"
+  >>> synthonspace.ReadTextFile(textFile)
+  >>> print(f"Number of reactions : {synthonspace.GetNumReactions()}")
+  Number of reactions : 1
+  >>> print(f"Number of products : {synthonspace.GetNumProducts()}")
+  Number of products : 12
+
+However, a large amount of pre-processing is required before the data are searchable which can be time consuming.
+For example, preparing the current REAL database (2024-09) of 70 billion compounds can take several hours.  It is
+therefore strongly recommended that the binary database is created before use.
+
+The fingerprints are written to the DB file, which means that the file must be searched using exactly the same type
+in order to get meaningful results.  This will be checked at runtime.  You may need to keep several versions of the
+database, labelled accordingly.
+
+.. doctest::
+
+  >>> fpgen = rdFingerprintGenerator.GetRDKitFPGenerator(fpSize=2048)
+  >>> synthonspace.BuildSynthonFingerprints(fpgen)
+  >>> dbFile = "amide_library_rdkit.spc"
+  >>> synthonspace.WriteDBFile(dbFile)
+
+There is also a convenience function for performing the conversion in one go:
+
+.. doctest::
+
+  >>> fpgen = rdFingerprintGenerator.GetMorganGenerator(fpSize=2048)
+  >>> rdSynthonSpaceSearch.ConvertTextToDBFile(textFile, dbFile, fpgen)
+
+although under the hood it does exactly the same steps.
+
+The binary database is read using the appropriate function:
+
+.. doctest::
+
+  >>> synthonspace.ReadDBFile(f"{RDBASE}/Code/GraphMol/SynthonSpaceSearch/data/idorsia_toy_space_a.spc")
+
+
+Memory Usage
+============
+
+The 2024-09 REAL database requires about 10GiB of RAM, FreedomSpace 2024-09 about 3.5GiB.  They should both therefore
```

Comment:
```suggestion
The 2024-09 REAL database requires about 10GB of RAM, FreedomSpace 2024-09 about 3.5GB.  They should both therefore
```

Reply:
I'm not too fussed either way given the extreme approximate rounding, but the OS does report GiB rather than GB.